aspnetcore/blazor/components.md

```diff
@@ -513,25 +513,27 @@ Prefer the strongly typed `EventCallback<T>` over `EventCallback`. `EventCallbac
 
 ## Capture references to components
 
-Component references provide a way to reference a component instance so that you can issue commands to that instance, such as `Show` or `Reset`. To capture a component reference, add a [@ref](xref:mvc/views/razor#ref) attribute to the child component and then define a field with the same name and the same type as the child component.
+Component references provide a way to reference a component instance so that you can issue commands to that instance, such as `Show` or `Reset`.
+
+The Razor compiler automatically generates a backing field for element and component references when using [@ref](xref:mvc/views/razor#ref). In the following example, there's no need to create a `myLoginDialog` field for the `LoginDialog` component:
```

Comment:
@guardrex, we're reverting automatic backing field generation. @NTaylorMullen is there an issue filed for doing that?

Reply:
Interesting ... I just spoke with Ryan offline yesterday about it, and he didn't mention it. Also, Dan's blog post goes this way. I *definitely* need guidance on how to handle it here.

Reply:
Yes @ajaybhargavb is correct. Unfortunately there are **fundamental** problems with automatic field generation that will prevent us from shipping it in 3.0

Reply:
@rynowak How do you want to handle it in docs? Stick with our Pre7 guidance to code the backing fields? ... i.e., revert these backing field language/examples in this PR?